Archivary/MAIN FORMS/FORM_SIDEBAR.cs

```diff
@@ -30,46 +31,156 @@ private void FORM_SIDEBAR_Load(object sender, EventArgs e)
 
         }
 
-        private void button1_Click(object sender, EventArgs e)
-        {
-            FormsRoot.loadParentForm(new FORM_LOGIN(FormsRoot));
-        }
-
-        private void button2_Click(object sender, EventArgs e)
-        {
-            isToggled = !isToggled;
-            collapseSidebar();
-        }
 
+        //
+        // SIDEBAR BUTTON
+        //
         private void collapseSidebar()
         {
             if (isToggled)
             {
-                logoPictureBox.Image = global::Archivary.Properties.Resources.ArchivaryLogoGreen;
-                PANEL_SIDEBAR.Size = new System.Drawing.Size(80, 1080);
+                PANEL_SIDEBAR.Size = new System.Drawing.Size(320, 1080);
+                dashboardLabel.Visible = true;
+                sidebarButton.ButtonImage = global::Archivary.Properties.Resources.ICON_SIDEBAR_CLOSE;
+                logoPictureBox.Image = global::Archivary.Properties.Resources.ArchivaryLogoBannerPNG;
             }
             else
             {
-                logoPictureBox.Image = global::Archivary.Properties.Resources.ArchivaryLogoBannerPNG;
-                PANEL_SIDEBAR.Size = new System.Drawing.Size(320, 1080);
+                PANEL_SIDEBAR.Size = new System.Drawing.Size(80, 1080);
```

Comment:
set mo siya 60 instead of 80 here, mas lumaki yung logo e, and nagpapakita yung titles unnecessarily
![image](https://github.com/Shinitaii/archivary/assets/101074825/ac238d8e-a389-4d73-b9f7-86103cd9c449)